helm/defectdojo/templates/django-ingress.yaml

```diff
@@ -29,9 +29,9 @@ metadata:
 {{- end }}
 spec:
 {{- if semverCompare ">=1.19.0" .Capabilities.KubeVersion.GitVersion }}
-   {{- if .Values.django.ingress.ingressClassName }}
-   ingressClassName: {{ .Values.django.ingress.ingressClassName }}
-   {{- end }}
+  {{- if .Values.django.ingress.ingressClassName }}
```

Comment:
I have no idea why this is marked as a change. It is exactly what was in the merged PR https://github.com/DefectDojo/django-DefectDojo/pull/6140